client/src/pages/CreateSurvey/index.jsx

```diff
@@ -1,7 +1,81 @@
-import React from "react";
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { TextField, Button, Switch } from '@material-ui/core';
 
 const CreateSurvey = () => {
-    return <></>;
+  const dispatch = useDispatch();
+  const setMetadata = (event, inputType) => {
+    const payload = {};
+    payload[event.target.name] =
+      inputType === 'switch' ? event.target.checked : event.target.value;
+    dispatch({ type: 'SET_METADATA', payload });
+  };
+  const {
+    title,
+    description,
+    recipients,
+    disclaimer,
+    anonymous,
+    // eslint-disable-next-line
+    questions,
```

Comment:
to not require the // eslint-disable-next-line, the trailing comma on questions needs to be removed.